lib/utils/tls.go

```diff
@@ -31,7 +38,13 @@ import (
 	cfsslhelpers "github.com/cloudflare/cfssl/helpers"
 	dockerarchive "github.com/docker/docker/pkg/archive"
 	"github.com/gravitational/license/authority"
+	teleutils "github.com/gravitational/teleport/lib/utils"
 	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	SelfSignedCertOrg = "Gravitational"
```

Comment:
I don't know what the best bet is here. I don't really have an issue with gravitational. The only advantage I see with using something else, is when looking at the certification and/or trying to search the sources it might be easier to locate if it was a string that isn't so common. So an org like "Gravitational Self-Signed Web Access" would create a distinction if someone looks at the cert, and is also easier to grep out of the code base.